docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,574 @@
+- `ca.name` - Give the CA a name by editing the parameter.
+- `csr.hosts` - Update this parameter to include this hostname and ip address where this server is running, if it is different than what is already in this file.
+- `port` - Enter the port that you want to use for this server.
+- `operations.listenAddress:` -  In the unlikely case that there is another node running on this host and port, then you need to update this parameter to use a different port.
+
+### Delete the TLS CA server certificates
+
+Before starting the server, if you modified the `csr` values or `hf.` attributes in the configuration .yaml file, you need to delete the `fabric-ca-server-tls/ca-cert.pem` file and the entire `fabric-ca-server-tls/msp` folder. These certificates will be re-generated when you start the CA server in the next step.
+
+### Start the TLS CA server
+
+Run the following command to start the CA server:
+```
+./fabric-ca-server start
+```
+When the server starts successfully you will see something similar to:
+```
+[INFO] Listening on https://0.0.0.0:7054
+```
+Because you have enabled TLS communications, notice that the TLS signing certificate `tls-cert.pem` file is generated under the `FABRIC_CA_HOME` location.
+
+**Tip:** The CA `ADMIN_USER` and `ADMIN_PWD` that were set on the `init` command cannot be overridden with the `-b` flag on this `start` command. When you need to modify the CA admin password, use the Fabric CA client [identity](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html#identity-command) command.
+
+**Optional flags**:
+- `-d` - If you want to run the server in DEBUG mode which facilitates problem diagnosis, you can include the `-d` flag on the start command. However, in general it is not recommended to run a server with debug enabled as this will cause the server to perform slower.
+- `-p` - If you want the server to run on a port different than what is specified in the configuration .yaml file, you can override the existing port.
+
+### Enroll node admins with TLS CA
+
+Now that your TLS CA is configured and before you can deploy any other nodes for your organization, you need to enroll the admin user of that node with the TLS CA to generate the TLS certificates. Since the CA server is up and running we instead of the using the **Fabric CA server CLI commands** we now use the **Fabric CA client CLI commands** to submit an enrollment request to the server. Before you can deploy another new CA, you need to provide the CA admin TLS certificate and private key to the configuration which allows the CA to transact security with the other nodes in the network. The following process is described for a CA node, but can be easily extended for peer or ordering nodes.
```

Comment:
Prior to enrolling node admins with the TLS CA, they need to be registered.  This can be done in one of two ways; 1) By adding node admins to the user registry section of the fabric-ca-server-config.yaml file (post 'init', and prior to restarting the server) or, 2) By manually registering the node admins from the command line with `fabric-ca-client register` after exporting the client home path and tls certfile location (you may also pass these as flags in the register command itself instead of exporting these variables first).

Reply:
I actually went with the option to manually register the node admins from the Fabric CA Client command line. I felt like that could be a more natural flow - if they register admins later after standing up the CA, which could be the case if intermediate CAs are added later.

Thanks for surfacing this important nuance.